production-swarm/docker-stack.yml

```diff
@@ -0,0 +1,58 @@
+version: '3'
+services:
+  jobservice:
+    env_file:
+      - ./rabbitmq.env
+      - ./postgres.env
+      - ./jobservice.env
+    environment:
+      CAF_DATABASE_USERNAME: "${CAF_DATABASE_USERNAME:-postgres}"
```

Comment:
Are these acceptable defaults for a postgres production environment? They could be but I'm just checking. If not they should probably be left configurable with no default given the proposed standard Andy sent out. Just to add these are not consistent with the marathon template which assumes no acceptable default.